3p/iframe-messaging-client.js

```diff
@@ -28,8 +28,9 @@ import {getMode} from '../src/mode';
 export class IframeMessagingClient {
   /**
    *  @param {!Window} win A window object.
+   *  @param {boolean=} broadcast whether to enable broadcast mode.
    */
-  constructor(win) {
+  constructor(win, broadcast) {
```

Comment:
I was thinking of having `IframeMessagingClient` operating in two different modes, controlled by constructor.

1. Caller pass in an optional `hostWin` param in constructor to run as the current mode.
1. Caller can omit the `hostWin` param to turn it into a `findHostWinYourSelf` mode.

This leads to better encapsulation, so that "broadcast" can be hidden from caller as an implementation detail.

Also, with that, we can kill the `setHostWindow` API.

Reply:
Sounds good. Done.